lib/carma/models/submission/metadata.rb

```diff
@@ -4,12 +4,42 @@ module CARMA
   module Models
     class Submission
       class Metadata < CARMA::Models::Base
-        request_payload_key :claim_id
+        request_payload_key :claim_id,
+                            :veteran,
+                            :primary_caregiver,
+                            :secondary_caregiver_one,
+                            :secondary_caregiver_two
 
-        attr_reader :claim_id
+        attr_accessor       :claim_id
+
+        attr_reader         :veteran,
+                            :primary_caregiver,
+                            :secondary_caregiver_one,
+                            :secondary_caregiver_two
 
         def initialize(args = {})
           @claim_id = args[:claim_id]
+
+          self.veteran = args[:veteran] || {}
+          self.primary_caregiver = args[:primary_caregiver] || {}
+          self.secondary_caregiver_one = args[:secondary_caregiver_one] if args[:secondary_caregiver_one]
+          self.secondary_caregiver_two = args[:secondary_caregiver_two] if args[:secondary_caregiver_two]
+        end
+
+        def veteran=(args = {})
```

Comment:
I think we should avoid using the name `args`.  It doesn't do a good job of describing what the arguments for the method are.  In the case of an initializer sometimes arguments are just a grab bag and "args" can make sense, but here it should be something like "veteran_data" or "veteran_hash"

Reply:
Also, default values for args doesn't make sense for an `=` method, so I would delete the ` = {}`